torchvision/models/detection/generalized_rcnn.py

```diff
@@ -77,6 +77,14 @@ def forward(self, images, targets=None):
             original_image_sizes.append((val[0], val[1]))
 
         images, targets = self.transform(images, targets)
+        if targets is not None:
+            for target in targets:
```

Comment:
Can you add some comments explaining that this is a check for degenerate boxes? In the future we might want to move this into a function (but no need to do it in this PR)